### PR TITLE
[ZEPPELIN-3195] Remove the limit on the number of run paragraphs at cron

### PR DESCRIPTION
**Pull request by mebelousov**
### What is this PR for?
Increase interpreter pool size for unlimit the number of paragraph that run at cron schedule.

### What type of PR is it?
[Hot Fix]

### What is the Jira issue?
*https://issues.apache.org/jira/browse/ZEPPELIN-3195

### How should this be tested?
* Create note with 15 paragraphs
* Check that at cron will be executed only 10 paragraphs
* Checkout this branch and rebuild app
* Check that all paragraphs are executed at cron

### Questions:
* Does the licenses files need update? NA
* Is there breaking changes for older versions? NA
* Does this needs documentation? NA

### CODE REVIEW

**Comment by zjffdu** [comment]
Actually, you can change the configuration in `zeppelin-site.xml`,  and change it to 65535 seems too large for me.

**Comment by mebelousov** [comment]
It's not easy to understand that "ZEPPELIN_INTERPRETER_MAX_POOL_SIZE" affects the number of paragraphs executed at cron. I would be glad if nobody will meet this bug and no need to change zeppelin-site.xml to fix note scheduling.

@zjffdu which number will be more appropriate? =)

**Comment by mebelousov** [comment]
@felixcheung 
Where I could document about ZEPPELIN_INTERPRETER_MAX_POOL_SIZE?
There is no such cases in documentation.

Also I cannot imagine than Zeppelin administrator will go to documentation when users will complain that manual note run is OK but only 10 first paragraphs are done at cron.

**Comment by mebelousov** [comment]
I renewed docs: remove old "ZEPPELIN_INTERPRETERS" and added ZEPPELIN_INTERPRETER_MAX_POOL_SIZE.
The default value for ZEPPELIN_INTERPRETER_MAX_POOL_SIZE remains 10.

File: docs/setup/operation/configuration.md [hunk 1]
```diff
@@ -281,6 +269,12 @@ If both are defined, then the **environment variables** will take priority.
     <td>http://repo1.maven.org/maven2/</td>
     <td>Remote principal repository for interpreter's additional dependency loading</td>
   </tr>
+  <tr>
+    <td><h6 class="properties">ZEPPELIN_INTERPRETER_MAX_POOL_SIZE</h6></td>
+    <td><h6 class="properties">zeppelin.interpreter.max.poolsize</h6></td>
+    <td>10</td>
+    <td>The maximum number of paragraphs that will be run at cron.</td>
```
Comment (zjffdu) [inline docs/setup/operation/configuration.md]: This doesn't sound like a correct description. `zeppelin.interpreter.max.poolsize` is for the number threads of per session configuration. That means if you are using 20 different interpreters, they can run simultaneously without affecting each other.

Comment (mebelousov) [reply docs/setup/operation/configuration.md]: I took note with 4 jdbc paragraph and a lof of python paragraphs.
You can see that only first 10 paragraphs were done.
first10

**Comment by mebelousov** [comment]
@zjffdu @felixcheung 
What will we do with this bug?

File: docs/setup/operation/configuration.md (hunk 1 above)
Comment (zjffdu) [reply docs/setup/operation/configuration.md]: @mebelousov This might be fixed in ZEPPELIN-3171, could you try that ?

Comment (mebelousov) [reply docs/setup/operation/configuration.md]: Year! This is fixed.
@zjffdu thank you!

**Pull request closed without merging**